### ISSUE TITLE
EmuHack-Open Door-battleGround (BB #294)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#294]
This issue was migrated from bitbucket.
**Original Reporter:** 
**Original Date:** 15.09.2010 14:14:41 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294

<hr>

When players use Emu hack for open door in all Battlaground.
The Door opens and closes ,but all Battleground is destroy.

And Test this bug.
Please Fix this bug.
Thx Oregon Team

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** pdx15
**Original Date:** 15.09.2010 14:29:23 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-245040

<hr>

Doors can be opened any key of type http:// www.wowhead.com/item=12942.
You spread a key on the panel, you approach to the door, you direct a mouse at a door, simultaneously you press a mouse on a door and the button on a key, any door opens.
As in the same way it is possible to clean GameObjects, for example mail.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LilleCarl
**Original Date:** 25.09.2010 05:55:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-255139

<hr>

This is kinda urgent i think so player stfx take a look at it :D

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** antoma
**Original Date:** 27.04.2013 09:56:57 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-3891868

<hr>

us remeber this bug,  few door in BG \ Arena not have flags=4 
group id and add flags

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** release_ok
**Original Date:** 04.01.2014 09:11:47 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-7584764

<hr>

Yeah, antoma is right. The easiest way to prevent this bug is to add `non-targetable` flag for thos doors:

```
#!sql

UPDATE `gameobject_template` SET `flags`='4' WHERE `entry` IN (id1, id2, id3);
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Yooki
**Original Date:** 04.01.2014 13:31:22 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-7586268

<hr>

Dirty, will you make a commit including a fix for this issue? I would get on it, but I don't have any knowledge about this particular problem.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** release_ok
**Original Date:** 04.01.2014 14:26:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-7586734

<hr>

I'm too lazy to download core on my PC.
You can make pull-request with this code and I'll merge it into the core.
.sql update should look like this:

```
#!sql

UPDATE `gameobject_template` SET `flags`='4' WHERE `entry` IN (179918, 179919, 179920, 179921, 180322, 179916, 179917, 184719, 184720, 180255, 180256);
```

This is only for Arathi Basin, Warsong Gulch and Eye of the Storm BGs

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Yooki
**Original Date:** 05.01.2014 00:00:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-7590743

<hr>

I hope you don't mind me checking that out before doing such a thing...
Would say I don't feel comfortable doing commits before I can back them up with some explanation and at least a weeks worth guarantee of it's functionality (no refunds though).

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 19.01.2014 23:54:14 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-7813046

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 17.02.2014 23:40:18 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-8442808

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 18.02.2014 02:01:11 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-8444020

<hr>

Fixed issue #294 - Activating GameObjects via EmuHack

→ <<cset 23ea72dd4374>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 18.02.2014 03:35:59 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/294/#comment-8444737

<hr>

Issue #363 was marked as a duplicate of this issue.
